quicktests/overlaying/tests/functional/doubleAxes.js

```diff
@@ -21,7 +21,12 @@ function run(svg, data, Plottable) {
   var plot2 = new Plottable.Plots.Line();
   plot2.addDataset(ds2);
   plot2.x(function(d) { return d.x; }, xScale2).y(function(d) { return d.y; }, yScale);
-  var plots = new Plottable.Components.Group([plot1, plot2]);
+
+  var dll = new Plottable.Components.DragLineLayer("horizontal")
```

Comment:
It might be better to have `DragLineLayer` in an example where there's also a `PanZoom` on one of the axes -- this allows us to test its updating logic (that it keeps the value it was dragged to when the `Scale` updates). Alternatively, have a `PanZoom` on the `Plot` and a `DragLineLayer` on the `Axis` (optionally linked with a `GuideLineLayer` in the main area).